kuma/users/views.py

```diff
@@ -289,6 +295,17 @@ def revision_by_distinct_doc(list_of_revisions):
     return [documents[doc_id] for doc_id in sorted(documents)]
 
 
+def ban_and_revert_notification(spammer, moderator, info):
+    subject = u'[MDN] %s has been banned by %s' % (spammer, moderator)
+    context = {'spammer': spammer,
+               'moderator': moderator}
+    context.update(info)
```

Comment:
Also, the `info` being passed in is the `context` variable of the caller, and I think it would be undesirable to mutate it.